README.md

```diff
@@ -55,4 +55,4 @@ htpasswd -b -c /etc/httpd/conf.d/starrs-auth.db root admin
 
 ```
 service httpd restart
-```
\ No newline at end of file
+```
```

Comment:
Why is there a diff here? Can you remove it if there isn't supposed to be a change here?